frigate/events.py

```diff
@@ -38,14 +43,35 @@ def run(self):
 
             logger.debug(f"Event received: {event_type} {camera} {event_data['id']}")
 
+            event_config: EventsConfig = self.config.cameras[camera].record.events
+
             if event_type == "start":
                 self.events_in_process[event_data["id"]] = event_data
 
-            if event_type == "end":
-                event_config: EventsConfig = self.config.cameras[camera].record.events
+            elif event_type == "update":
+                self.events_in_process[event_data["id"]] = event_data
+                # TODO: this will generate a lot of db activity possibly
+                if event_data["has_clip"] or event_data["has_snapshot"]:
+                    Event.replace(
+                        id=event_data["id"],
+                        label=event_data["label"],
+                        camera=camera,
+                        start_time=event_data["start_time"] - event_config.pre_capture,
+                        end_time=None,
```

Comment:
end_time can not be None/Null, peewee comes back with a database exception, I believe you need to set DateTimeField(null=True) in models.py:Event()

Reply:
There is a database migration that should be removing the null constraint. I am not seeing any issues here.

Reply:
Yep, a problem with my adhoc dev environment. I will follow the contributing guidelines from now on.